lib/msal-node/src/network/LoopbackClient.ts

```diff
@@ -119,8 +94,17 @@ export class LoopbackClient implements ILoopbackClient {
      * Close the loopback server
      */
     closeServer(): void {
-        if (!!this.server) {
+        if (this.server) {
+            // Only stops accepting new connections, server will close once open/idle connections are closed.
             this.server.close();
+
+            if (typeof this.server.closeAllConnections === "function") {
+                /*
+                 * Close open/idle connections. This API is available in Node versions 18.2 and higher
+                 */
+                this.server.closeAllConnections();
```

Comment:
if you wanted to handle it below 18.2 you could listen for "connection" events to get a reference to the socket connections, though perhaps that is more bookkeeping than you want to do. Another way to resolve this trouble is you can call `unref` on the server object to prevent it from keeping the Node runtime alive, I think that might be sufficient for our purposes.

```suggestion
                this.server.closeAllConnections();
                this.server.unref();
```

Reply:
whoops, the suggestion should be calling unref outside of this if block for downlevel runtimes to benefit, but you get the idea :)